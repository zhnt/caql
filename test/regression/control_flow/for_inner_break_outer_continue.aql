// Nested loops: inner break, outer continue
let sum = 0
for i = 1, 4, 1 {
    if i == 3 { continue }
    for j = 1, 4, 1 {
        if j == 3 { break }
        sum = sum + i * 10 + j
        print(i * 10 + j)
    }
}
print(sum)
