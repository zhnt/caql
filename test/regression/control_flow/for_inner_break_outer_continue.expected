11
12
21
22
66
